internal/third_party/pkgsite/source/source.go

```diff
@@ -443,20 +465,39 @@ func matchLegacyTemplates(ctx context.Context, sm *sourceMeta) (_ urlTemplates,
 // subdirectories. See https://research.swtch.com/vgo-module for a discussion of
 // the "major branch" vs. "major subdirectory" conventions for organizing a
 // repo.
-func adjustVersionedModuleDirectory(ctx context.Context, client *Client, info *Info) {
+//
+// !! MODIFIED FOR google/go-licenses use-cases.
+func adjustVersionedModuleDirectory(ctx context.Context, client *Client, info *Info) error {
 	dirWithoutVersion := removeVersionSuffix(info.moduleDir)
 	if info.moduleDir == dirWithoutVersion {
-		return
+		return nil
 	}
+
+	// prefer querying the raw URL, this URL generally has higher rate limits
```

Comment:
During experimentation, the raw endpoint performed noticeably better (I did not receive any 429 responses).